packages/core/components/ListPicker/index.tsx

```diff
@@ -120,38 +121,42 @@ export default function ListPicker(props: ListPickerProps) {
                 />
                 <div className={styles.buttons}>
                     {onSelectAll && (
+                        <Tooltip content={hasUnselectedItem ? undefined : "All options selected"}>
+                            <ActionButton
+                                ariaLabel="Select all"
+                                className={classNames(
+                                    {
+                                        [styles.disabled]: !hasUnselectedItem,
+                                    },
+                                    styles.selectAllButton,
+                                    styles.actionButton
+                                )}
+                                disabled={!hasUnselectedItem}
+                                iconProps={{ iconName: "MultiSelect" }}
+                                title={hasUnselectedItem ? undefined : "All options selected"}
```

Comment:
Spotting several instances of title still but in other replaces removed; is the tooltip component intended to be a direct replacement?

Reply:
Ah I seem to have missed a couple, should be caught in https://github.com/AllenInstitute/biofile-finder/pull/241/commits/056a3a915788837a9750eaa5a80191919d29f503, but lmk if you are seeing more

Reply:
I think the only other one I'm seeing is in ListRow.tsx